code/game/gamemodes/umbrage/umbrage_abilities/silver_tongue.dm

```diff
@@ -0,0 +1,54 @@
+//Can be used on a communications console to recall the shuttle. Leaves visible evidence.
+/datum/action/innate/umbrage/silver_tongue
+	name = "Silver Tongue"
+	id = "silver_tongue"
+	desc = "When used near a communications console, allows you to forcefully transmit a message to Central Command, initiating a shuttle recall."
+	button_icon_state = "umbrage_silver_tongue"
+	check_flags = AB_CHECK_CONSCIOUS|AB_CHECK_STUNNED
+	psi_cost = 100
+	lucidity_cost = 1 //Very niche, so low cost
+	blacklisted = 0
+
+/datum/action/innate/umbrage/silver_tongue/IsAvailable()
+	if(SSshuttle.emergency.mode != SHUTTLE_CALL)
+		return
+	return ..()
+
+/datum/action/innate/umbrage/silver_tongue/Activate()
+	var/obj/machinery/computer/communications/C = locate() in range(1, owner)
+	if(!C)
+		owner << "<span class='warning'>There are no communications consoles nearby.</span>"
+		return
+	if(C.stat)
+		owner << "<span class='warning'>[C] is depowered.</span>"
+		return
+	owner.visible_message("<span class='warning'>[owner] presses \his hand against [src]'s keys, and they begin to move by themselves!</span>", \
+	"<span class='velvet bold'>[pick("Oknnu. Pda ywlpwej swo hkccaz ej.", "Pda aiancajyu eo kran. Oknnu bkn swopejc ukqn peia.", "We swo knzanaz xu Hws Psk. Whh ckkz jks.")]</span>\n\
+	<span class='notice'>You begin transmitting a recall message to Central Command...</span>")
+	play_recall_sounds(C)
+	if(!do_after(owner, 30, target = C))
```

Comment:
I don't mind that so much. The canon behind it is that you give the computer a jolt to press buttons by itself, than stick by it to confirm the recall once it finishes.